### DISCUSSION TITLE
Regarding Credits of Certificate

### DISCUSSION THREAD

**Discussion by mehboobpatel** [github/india#829] [General]
Hello official

    My question is when we receive the credited certificate so how can we tell the college its credited and qualified for credits since our internship semester is in December 2022 so how can we show this certificate for last internship as there is no where mentioned in the certificate that it is eligible for Credits

**Comment by mehboobpatel** [comment]
Please answer
@ey-mentor1 @ey-mentor2 @EY-Mentor3 @EY-Mentor4

**Comment by ey-mentor1** [comment]
Hello,
The certificate is created on the date when the mentors evaluates your project as approved.
We cannot share certificates on back date 
Thank you
